source/features/tags-on-commits-list.tsx

```diff
@@ -141,12 +139,15 @@ async function init(): Promise<void | false> {
 	await cache.set(cacheKey, cached, {days: 1});
```

Comment:
To avoid race conditions, we need to split this up to cache each commit individually calling `cache.set` as needed, for example replacing `commitsWithNoTags.push(targetCommit);` with `void cache.set('${cacheKey}:${commit}', [])`

Reply:
> What does this means here? Isn't per commit has been cached now?

Yeah you're right, it's just that due to the querying it has limited utility.